Xamarin.Forms.Build.Tasks/PropertyDefinitionExtensions.cs

```diff
@@ -5,11 +5,17 @@ namespace Xamarin.Forms.Build.Tasks
 {
 	static class PropertyDefinitionExtensions
 	{
-		public static TypeReference ResolveGenericPropertyType(this PropertyDefinition self, TypeReference declaringTypeReference)
+		public static TypeReference ResolveGenericPropertyType(this PropertyDefinition self, TypeReference declaringTypeRef,
+			ModuleDefinition module)
```

Comment:
most of the extension methods are passing the module. I'm quite sure one day a special case will require the module to be there.

anyway, the cost at compile time is negligible, and there's no runtime implication